src/event.rs

```diff
@@ -446,51 +446,55 @@ impl Clone for WindowEvent<'static> {
 }
 
 impl<'a> WindowEvent<'a> {
-    pub fn to_static(self) -> Option<WindowEvent<'static>> {
+    pub fn to_static(&self) -> Option<WindowEvent<'static>> {
```

Comment:
I think this is a breaking change, and hence should be noted in the changelog. See also https://github.com/rust-windowing/winit/issues/1968.

I would prefer having this fixed in a separate PR

Reply:
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

Reply:
That was committed by mistake will revert it.